config.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+function pre_build {
+    echo "Hello world (this should not be run)"
+}
+
+function run_tests {
+    echo "Hello world (this should not be run)"
+}
+
```

Comment:
without these, multibuild complains (we are only using it to install things on osx, that's why these functions are not actually run)